.github/workflows/nvfuser-ci-trigger.yml

```diff
@@ -43,6 +43,7 @@ jobs:
             github.actor == 'cowanmeg' ||
             github.actor == 'nsarka' ||
             github.actor == 'syed-ahmed' ||
+            github.actor == 'mattw' ||
```

Comment:
this needs to be the same as your github handle, not internal one

Reply:
AI successfully caught that to my surprise! https://github.com/NVIDIA/Fuser/pull/4901#issuecomment-3145995430

Reply:
:exploding_head: